augly/tests/image_tests/transforms_unit_test.py

```diff
@@ -25,6 +25,9 @@ def test_Blur(self):
 
     def test_Brightness(self):
         self.evaluate_class(imaugs.Brightness(), fname="brightness")
+    
+    def test_Skew(self):
+        self.evaluate_class(imaugs.Skew(), fname="skew")
```

Comment:
@tanujdhiman when you run this test, you'll see that you also need to modify the file `augly/utils/expected_output/image/expected_metadata.json` by adding a dict of the metadata expected after running this augmentation for "skew".

Reply:
Yes, I've updated this too.

Thanks